plugin/src/main/scala/migrate/TypeInferenceMigration.scala

```diff
@@ -1,43 +1,49 @@
 package migrate
 
 import migrate.interfaces.CompilationException
-import ScalaMigratePlugin.{ migrateAPI, inputsStore, scala3Version, Keys }
-
+import ScalaMigratePlugin.{ Keys, inputsStore, migrateAPI, scala3Version }
+import migrate.TypeInferenceMigration.{ errorMessage, successMessage }
 import sbt.Keys._
 import sbt._
 
 import scala.io.AnsiColor._
-import scala.util.{ Try, Success, Failure }
+import scala.util.{ Failure, Success, Try }
 import scala.collection.JavaConverters._
-
 import java.nio.file.Files
 
 private[migrate] object TypeInferenceMigration {
   val internalImpl = Def.taskDyn {
-    val projectRef = thisProjectRef.value
-    val configs    = Keys.migrationConfigs.value
-    val sv         = scalaVersion.value
-
-    val projectId = projectRef.project
+    implicit val projectRef = thisProjectRef.value
+    val configs             = Keys.migrationConfigs.value
+    val sv                  = scalaVersion.value
+    implicit val projectId  = projectRef.project
 
```

Comment:
This has nothing to do with the PR, but I really don't like that the formatter aligns everything. It adds a lot of changes that are not real changes.

Reply:
maybe the implicits ?